admin/spec/service/pin.service.spec.js

```diff
@@ -0,0 +1,116 @@
+const proxyquire = require('proxyquire').noCallThru()
+const sinon = require('sinon')
+const moment = require('moment')
+const pupilDataService = require('../../services/data-access/pupil.data.service')
+const schoolDataService = require('../../services/data-access/school.data.service')
+const checkDataService = require('../../services/data-access/check.data.service')
+const pinService = require('../../services/pin.service')
+const jwtService = require('../../services/jwt.service')
+const pinValidator = require('../../lib/validator/pin-validator')
+const pupilMock = require('../mocks/pupil')
+const schoolMock = require('../mocks/school')
+
+/* global describe, it, expect, beforeEach, afterEach, spyOn */
+
+describe('pin.service', () => {
+  let sandbox
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create()
+  })
+
+  afterEach(() => sandbox.restore())
+  describe('getPupilsWithActivePins', () => {
+    let sandbox
+    let pupil1
+    let pupil2
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create()
```

Comment:
sandbox has already been created - this is duplication

Reply:
@jon-shipley thanks I've updated that